### PR TITLE
Vacms 16144 VAMC service location paragraphs - and merge VBA

### PR DESCRIPTION
**Pull request by eselkin**
# DO NOT MERGE - DEPENDS ON CMS PR-[15622](https://github.com/department-of-veterans-affairs/va.gov-cms/pull/15622/files)

## Summary

- Takes existing VAMC service location paragraph, adds appointment information. Removes VBA service location paragraph and merges content. Reorganizes both to follow VBA order from designs. Improves reasons for showing or not showing content on the component.
- Sitewide Facilities
- Integration branch for VAMC, but coordinates with VBA
 
## Related issue(s)

- relates to https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16144
- blocked by https://github.com/department-of-veterans-affairs/va.gov-cms/pull/15622

## Testing done

- VAMC service locations were just inline with the accordions but the paragraphs were difficult to parse and now more information is added to the paragraph types (including appointment info, etc). 
- VAMC still render as do VBA facilities for regional and facility services
- Added test for VAMC health service, non-clinical service, existing VBA service test this for compliance
 
## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._
<details>
<summary>Desktop non-clinical</summary>

</details>

<details>
<summary>Desktop clinical</summary>

</details>

<details>
<summary>Desktop VBA. Some fields missing from the VBA query to populate appointments</summary>

</details>

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

## Acceptance Criteria
- [x] Timing is coordinated with Drupal engineers
- [x] Query is updated to remove these fields from VAMC Facility Service (health_care_local_health_service) 
   - [x]  field_hservice_appt_intro_select
   - [x]  field_hservice_appt_leadin
   - [x] field_online_scheduling_availabl
   - [x] field_walk_ins_accepted
   - [x]  field_phone_numbers_paragraph
- [x] Query is updated to add the fields from the Service Location paragraph.
   - [x] Not sure what fields have already been added.  This will give you the [list of all the fields in the service location paragraph](https://pr15622-xptmfsnjoodldn85nzgkrgljqtfbz24b.ci.cms.va.gov/admin/reports/content-model/fields?field_name=&field_type=&entity_type=paragraph&bundle=service_location&label=)
- [x] Template is updated to remove the deleted fields and add the new fields.
  - [x] Walk-ins accepted is no longer showing.
  - [x] Service Delivery options are showing.
  - [x] For Appointments intro text, if Customize text is selected but no text is entered, nothing is shown (acts like Remove text was selected)
- [x] Accordion elements are confirmed
- [x] Merge the VBA and VAMC Facility and Regional Service Location paragraph type liquid templates.
- [x] This ticket need to go into their own integration branch. (This is the integration branch and has Do not merge on it)
- [x] Testing is completed as needed
- [ ] Design is reviewed (tag in PR)
- [ ] Accessibility is reviewed (tag in PR)

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

## Requested Feedback

1. view VAMC non-clinical services at: https://web-pr15622-xptmfsnjoodldn85nzgkrgljqtfbz24b.ci.cms.va.gov/fayetteville-arkansas-health-care/medical-records-office/

2. view VAMC clinical services at: https://web-pr15622-xptmfsnjoodldn85nzgkrgljqtfbz24b.ci.cms.va.gov/washington-dc-health-care/locations/washington-va-medical-center/

**Note: I am concerned about us displaying the "Appointments" header when there's really no appointment text like in Anesthesiology or Emergency Care**
 
3. view VBA services at: https://web-pr15622-xptmfsnjoodldn85nzgkrgljqtfbz24b.ci.cms.va.gov/cleveland-va-regional-benefit-office/locations/cleveland-va-regional-benefit-office/

### CODE REVIEW

File: src/site/paragraphs/tests/vamc/vamc_field_service_location.unit.spec.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
+/* eslint-disable @department-of-veterans-affairs/axe-check-required */
```
[the remark this answers is not in the document]
Comment (eselkin) [reply src/site/paragraphs/tests/vamc/vamc_field_service_location.unit.spec.js:1]: Not a cypress test

File: src/site/paragraphs/service_location.drupal.liquid [hunk 2]
```diff
@@ -1,62 +1,174 @@
...
+    possibly values for fieldOfficeVists
+      no
+      yes_appointment_only
+      yes_first_come_first_served_basis
+      yes_with_or_without_appointment
+  {% endcomment %}
+  {% if single.fieldOfficeVisits and single.fieldOfficeVisits != "no" %}
+    <div class="vads-u-margin-top--0p5" data-testid="service-location-field-office-visits">
+      <p>
+        <i class="fas fa-building vads-u-margin-right--0p5"></i>
```
[the remark this answers is not in the document]
Comment (eselkin) [reply src/site/paragraphs/service_location.drupal.liquid:25]: Requested by design

File: src/site/paragraphs/service_location.drupal.liquid [hunk 3]
```diff
@@ -25,38 +1,174 @@
...
+  {% comment %} 
+    possibly values for fieldVirtualSupport
+      value: 'no'
+      value: yes_appointment_only
+      value: yes_first_come_first_served_basis
+      value: yes_with_or_without_appointment
+  {% endcomment %}
+  {% if single.fieldVirtualSupport and single.fieldVirtualSupport != "no" %}
+    <div data-testid="service-location-virtual-support">
+      <p>
```
[the remark this answers is not in the document]
Comment (eselkin) [reply src/site/paragraphs/service_location.drupal.liquid:47]: Requested by design

File: src/site/paragraphs/service_location.drupal.liquid [hunk 4]
```diff
@@ -25,38 +1,174 @@
...
+      </p>
+    </div>
+  {% endif %}
+  
+
+  {% if fieldReferralRequired %}
+    <div>
+      <p>
+        {% case fieldReferralRequired %}
+          {% when "1","2" %}
```
[the remark this answers is not in the document]
Comment (eselkin) [reply src/site/paragraphs/service_location.drupal.liquid:58]: Requested by design

File: src/site/paragraphs/service_location.drupal.liquid [hunk 5]
```diff
@@ -25,38 +1,174 @@
...
+            <i class="fas fa-check-circle vads-u-margin-right--0p5"></i> A referral is required
+          {% when "not_applicable","0" %}
```
[the remark this answers is not in the document]
Comment (eselkin) [reply src/site/paragraphs/service_location.drupal.liquid:60]: Requested by design

File: src/site/paragraphs/service_location.drupal.liquid [hunk 6]
```diff
@@ -1,62 +1,174 @@
...
+    possibly values for fieldVirtualSupport
+      value: 'no'
+      value: yes_appointment_only
+      value: yes_first_come_first_served_basis
+      value: yes_with_or_without_appointment
+  {% endcomment %}
+  {% if single.fieldVirtualSupport and single.fieldVirtualSupport != "no" %}
+    <div data-testid="service-location-virtual-support">
+      <p>
+        <i class="fas fa-calendar vads-u-margin-right--0p5"></i> Schedule a virtual appointment
```
[the remark this answers is not in the document]
Comment (eselkin) [reply src/site/paragraphs/service_location.drupal.liquid:48]: Requested by design

File: src/site/paragraphs/service_location.drupal.liquid [hunk 7]
```diff
@@ -25,38 +1,174 @@
...
+      </p>
+    </div>
+  {% endif %}
+  
+
+  {% if fieldReferralRequired %}
+    <div>
+      <p>
+        {% case fieldReferralRequired %}
+          {% when "1","2" %}
```
[the remark this answers is not in the document]
Comment (eselkin) [reply src/site/paragraphs/service_location.drupal.liquid:58]: Requested by design

**Comment by eselkin** [comment]
@davidmpickett There's a concern I had when displaying the Appointments header. Certain service location paragraphs like the Emergency care department at the DC VAMC facility show `"fieldOfficeVisits": "yes_with_or_without_appointment"` which doesn't make sense but because they do it shows the Appointments header.

Also certain facilities show the appointments header because of the above issue but have no Appointments context within the section. Like no custom text even though they selected "customize_text". I'm wondering if we need a more complex condition on whether to show Appointments. Like all the conditions of your matrix plus whether customize text is selected and if there's actually text or if use_default_text is selected.

**Comment by eselkin** [comment]
@laflannery thanks for your fast feedback. 

The i tags will be updated. 

I was working on the conditional heading level stuff. I must have left it out in the commit. The VAMC accordions are awash with weird liquid stuff. Will fix. 

I can update the non-clinical services that don't have accordions in this PR.

**Comment by eselkin** [comment]
@laflannery I misinterpreted what you were saying when I first re-looked at this. The facility name (that has a link) should be in an h4 but the sections in the card/box should be at h5 level? I think that's what you're saying. We should double check that also works on other non-clinical pages. I'll link some later.

File: src/site/paragraphs/service_location.drupal.liquid [hunk 8]
```diff
@@ -1,62 +1,178 @@
...
+    possibly values for fieldOfficeVists
+      no
+      yes_appointment_only
+      yes_first_come_first_served_basis
+      yes_with_or_without_appointment
+  {% endcomment %}
+  {% if single.fieldOfficeVisits and single.fieldOfficeVisits != "no" %}
+    <div class="vads-u-margin-top--0p5" data-testid="service-location-field-office-visits">
+      <p>
+        <i class="fas fa-building vads-u-margin-right--0p5" aria-hidden="true"></i>
```
[the remark this answers is not in the document]
Comment (eselkin) [reply src/site/paragraphs/service_location.drupal.liquid:29]: requested in design

File: src/site/paragraphs/service_location.drupal.liquid [hunk 9]
```diff
@@ -1,62 +1,178 @@
...
+    </div>
+  {% endif %}
+  
+
+  {% if fieldReferralRequired %}
+    <div>
+      <p>
+        {% case fieldReferralRequired %}
+          {% when "1","2" %}
+            <i class="fas fa-check-circle vads-u-margin-right--0p5" aria-hidden="true"></i> A referral is required
```
[the remark this answers is not in the document]
Comment (eselkin) [reply src/site/paragraphs/service_location.drupal.liquid:63]: requested in design

File: src/site/paragraphs/service_location.drupal.liquid [hunk 10]
```diff
@@ -1,62 +1,178 @@
...
+          {% when "not_applicable","0" %}
+            <i class="fas fa-times-circle vads-u-margin-right--0p5" aria-hidden="true"></i> A referral is not required
```
[the remark this answers is not in the document]
Comment (eselkin) [reply src/site/paragraphs/service_location.drupal.liquid:65]: requested in deisn

File: src/site/paragraphs/service_location.drupal.liquid [hunk 11]
```diff
@@ -1,62 +1,178 @@
...
+    possibly values for fieldVirtualSupport
+      value: 'no'
+      value: yes_appointment_only
+      value: yes_first_come_first_served_basis
+      value: yes_with_or_without_appointment
+  {% endcomment %}
+  {% if single.fieldVirtualSupport and single.fieldVirtualSupport != "no" %}
+    <div data-testid="service-location-virtual-support">
+      <p>
+        <i class="fas fa-calendar vads-u-margin-right--0p5" aria-hidden="true"></i> Schedule a virtual appointment
```
[the remark this answers is not in the document]
Comment (eselkin) [reply src/site/paragraphs/service_location.drupal.liquid:52]: requested in design

**Comment by thejordanwood** [comment]
@eselkin I have some notes on some spacing inconsistencies. If the way I worded any of this is confusing then we can meet up to discuss if that works better. 

<details>
<summary>Desktop non-clinical</summary>

1. The spacing between the top and bottom of the card should be the same. The design has these at 16px, which I think is closer to what you have for the bottom spacing here. 


</details>

<details>
<summary>Desktop clinical</summary>

1. The "Nurse advice line" heading should be closer to the Main Phone section. It should have the same spacing as Main Phone does to the Appointments section.


2. Spacing below the Main Phone section is too big here as well. It should have the same spacing as the top. 


3. Can the spacing between the CTA and Audiology and Speech Clinic section be the same as the spacing between the CTA and Appointments section?


4. The spacing between the Main Phone section and Service Hours is a bit big here too. Also, the sentence "The service hours are the same..." has too much space from the Service Hours heading. It should be the same spacing as the phone number is to the Main Phone heading here. 

</details>

<details>
<summary>Desktop VBA</summary>

1. Spacing looks pretty good overall. Only minor thing I see is the sentence "The service hours are the same..." and Service Hours heading should be closer together. 



</details>

**Comment by eselkin** [comment]
@thejordanwood The problem with the spacing is that sometimes some sections appear and sometimes they don't because of the data. I'll work to see if we can make some sort of indicator for prior paragraphs. When we make test data we're always guaranteed that the data exists so it looks good like with VBA. But in the real data it's a little more chaotic.

**Comment by thejordanwood** [comment]
@eselkin I had a feeling that was probably the case. Thanks for looking into it!

**Comment by davidmpickett** [comment]
> @davidmpickett There's a concern I had when displaying the Appointments header. Certain service location paragraphs like the Emergency care department at the DC VAMC facility show `"fieldOfficeVisits": "yes_with_or_without_appointment"` which doesn't make sense but because they do it shows the Appointments header.
> 
> Also certain services show the appointments header because of the above issue but have no Appointments context within the section. Like no custom text even though they selected "customize_text". I'm wondering if we need a more complex condition on whether to show Appointments. Like all the conditions of your matrix plus whether customize text is selected and if there's actually text or if use_default_text is selected.

I might have been misinterpreting your original question. The matrix logic I mapped out applies to the Appointments Information as a whole. So appointment Header, intro text, appointment phone, online scheduling available. 

Appointments

In the 4 scenarios where an editor has indicated that appointments don't apply to this service location then none of the appointment information should display. All of it should be hidden. 

Logic for displaying Appointment Info on Front End

However, in the 12 scenarios in which appointments are a possibility, then each of the individual items should have it's own logic:
-  Online scheduleing link only shows if online scheduling set to true
- Appointment phone number(s) show if populated (it's possible to unselect use facility phone and not add any phone numbers)
- Appointment text should show if default or custom is selected (plus guard against empties)
- Appointment header should show if default or custom is selected (plus guard against empties)?

I'm still conflicted about the logic for removing appointment header. It is functioning as a pseudo-semantic element where it contextualizes the text, the phone number, and the link. So for instance if appointments are possible, but an editor chooses to remove the lead in text (or enters empty custom text) but still adds a phone number for appointments, but doesn't label that phone number as being for appointments we basically are choosing between two bad options. A header with no content under it or a phone number that doesn't give the user enough info to understand it's purpose 

Service Location 1

Service Location 2

Not sure if I cleared anything up, or just added to the confusion.

**Comment by davidmpickett** [comment]
> * The h5 headings within the Service accordions need to be conditional. If you look at [the Sketch](https://www.sketch.com/s/8bda90fd-99e8-4761-9210-3a24282eb811) It has "Optional service location title" assigned as the h4. However, because this is optional, it means the heading structure in Sketch is only applicable **IF** that title is present. You can see this as correct on the [VBA service accordions](https://web-pr15622-xptmfsnjoodldn85nzgkrgljqtfbz24b.ci.cms.va.gov/cleveland-va-regional-benefit-office/locations/cleveland-va-regional-benefit-office/). If the optional title is not present though, then all the h5s indicated in Sketch need to be h4s. You can see that currently on the [VAMC service accordions](https://web-pr15622-xptmfsnjoodldn85nzgkrgljqtfbz24b.ci.cms.va.gov/washington-dc-health-care/locations/washington-va-medical-center/) there is an incorrect heading structure because the h4 level is being skipped.
> * This might be out of scope so if it is, I can ticket it separately:

So we currently don't have any data/content model to support the "optional service location title" for VAMC services health or non-clinical). We probably shouldn't be building with an expectation of it being present until we at least have a plan for what the data source would be. This is likely a vestige of the VAMC accordion being based on the VBA accordion design, but somehow this slipped past the Refinement process

**Comment by eselkin** [comment]
> > * The h5 headings within the Service accordions need to be conditional. If you look at [the Sketch](https://www.sketch.com/s/8bda90fd-99e8-4761-9210-3a24282eb811) It has "Optional service location title" assigned as the h4. However, because this is optional, it means the heading structure in Sketch is only applicable **IF** that title is present. You can see this as correct on the [VBA service accordions](https://web-pr15622-xptmfsnjoodldn85nzgkrgljqtfbz24b.ci.cms.va.gov/cleveland-va-regional-benefit-office/locations/cleveland-va-regional-benefit-office/). If the optional title is not present though, then all the h5s indicated in Sketch need to be h4s. You can see that currently on the [VAMC service accordions](https://web-pr15622-xptmfsnjoodldn85nzgkrgljqtfbz24b.ci.cms.va.gov/washington-dc-health-care/locations/washington-va-medical-center/) there is an incorrect heading structure because the h4 level is being skipped.
> > * This might be out of scope so if it is, I can ticket it separately:
> 
> So we currently don't have any data/content model to support the "optional service location title" for VAMC services health or non-clinical). We probably shouldn't be building with an expectation of it being present until we at least have a plan for what the data source would be. This is likely a vestige of the VAMC accordion being based on the VBA accordion design, but somehow this slipped past the Refinement process

I think it's alright if it doesn't exist. The service location title/name @laflannery was talking about was outside the box not something inside the box there.

**Comment by laflannery** [comment]
@eselkin I took another look at the headings and on the [VAMC Clinical page](https://web-pr15622-xptmfsnjoodldn85nzgkrgljqtfbz24b.ci.cms.va.gov/washington-dc-health-care/locations/washington-va-medical-center/) I think there was a case that wasn't accounted for or wasn't clearly labelled in the Sketch designs.

There are phone numbers being added in the appointment section in Drupal and these numbers need to remain close to the "Appointments" heading in the new display so it is clear why a person would use this number. Here's an example of the new confusing scenario I'm looking at:
image

- The H5 Geriatric Service is actually for appointments:
image
- The H4 Geriatric Service and the Main Phone have been indicated in the general Contact Info
image

**Looking at the Sketch and the information provided I would recommend the following:**
- H4 "Appointments"
  - H5 Specific Appointments Phone label
    - Specific Appointments Phone number
  - Secondary CTA to Schedule an appointment
- H4 Specific Address label
  - Address info
- H4 "Main Phone"
  - Phone number
- H4 Specific Contact phone label
  - Specific Contact phone number
- H4 Specific Contact email label
  - Specific Contact email address
- H4 Specific Hours label
  - Hours info

@davidmpickett and/or @thejordanwood to also weight in

**Comment by davidmpickett** [comment]
> **Looking at the Sketch and the information provided I would recommend the following:**
> 
> * H4 "Appointments"
>   
>   * H5 Specific Appointments Phone label
>     
>     * Specific Appointments Phone number
>   * Secondary CTA to Schedule an appointment
> @davidmpickett and/or @thejordanwood to also weight in

I do think that @laflannery's suggestion of changing the semantics of the Appointments section makes sense. This could also help with some of the confusing logic of when to hide or show the Appointments header that I responded to above. But I also want to call out that these might qualify as product decisions, and/or be out of scope. 

Flagging @xiongjaneg @jilladams for un-blocking / scope assessment

File: src/site/paragraphs/service_location.drupal.liquid [hunk 12]
```diff
@@ -1,62 +1,190 @@
...
+      yes_first_come_first_served_basis
+      yes_with_or_without_appointment
+  {% endcomment %}
+  {% assign showIconDiv = single.fieldOfficeVisits | shouldShowIconDiv: single.fieldVirtualSupport, fieldReferralRequired  %}
+  {% if showIconDiv %}
+    <div class="vads-u-padding-top--1">
+      {% if single.fieldOfficeVisits and single.fieldOfficeVisits != "no" %}
+        <div data-testid="service-location-field-office-visits">
+          <p class="vads-u-margin-top--1 vads-u-margin-bottom--0">
+            <i class="fas fa-building vads-u-margin-right--0p5" aria-hidden="true"></i>
```
[the remark this answers is not in the document]
Comment (eselkin) [reply src/site/paragraphs/service_location.drupal.liquid:36]: required by design

File: src/site/paragraphs/service_location.drupal.liquid [hunk 13]
```diff
@@ -1,62 +1,190 @@
...
+        possibly values for fieldVirtualSupport
+          value: 'no'
+          value: yes_appointment_only
+          value: yes_first_come_first_served_basis
+          value: yes_with_or_without_appointment
+      {% endcomment %}
+      {% if single.fieldVirtualSupport and single.fieldVirtualSupport != "no" %}
+        <div data-testid="service-location-virtual-support">
+          <p class="vads-u-margin-top--1 vads-u-margin-bottom--0">
+            <i class="fas fa-calendar vads-u-margin-right--0p5" aria-hidden="true"></i> Schedule a virtual appointment
```
[the remark this answers is not in the document]
Comment (eselkin) [reply src/site/paragraphs/service_location.drupal.liquid:59]: required by design

File: src/site/paragraphs/service_location.drupal.liquid [hunk 14]
```diff
@@ -1,62 +1,190 @@
...
+      {% endif %}
+      
+
+      {% if fieldReferralRequired %}
+        <p class="vads-u-margin-top--1">
+          {% case fieldReferralRequired %}
+            {% when "1" %}
+              <i class="fas fa-check-circle vads-u-margin-right--0p5" aria-hidden="true"></i> A referral is required
+            {% when "0","not_applicable","2" %}
+              <i class="fas fa-times-circle vads-u-margin-right--0p5" aria-hidden="true"></i> A referral is not required
```
[the remark this answers is not in the document]
Comment (eselkin) [reply src/site/paragraphs/service_location.drupal.liquid:71]: required by design

**Comment by eselkin** [comment]
@laflannery I think I can pull up that 2nd geriatric up under appointments. It's odd and a bug that it's an h5 in there now, but it works for your updated organization.

Re @davidmpickett's comment I will wait on suggestions from @jilladams  and @xiongjaneg and will keep working on merging in the conflicts from the MH phone number stuff from Max.

**Comment by davidmpickett** [comment]
While poking around, I noticed a few other things

**Requested change to content:**
- When by appointment only is selected, the displayed text should be "Visit our office, ~~with an~~ by appointment only"

**Not matching the design**
- The accordions should be the borderless variant.
Screenshot 2024-02-26 090149

**This isn't good, but I don't know how to fix it**

- If an editor has entered `building name/number` or `wing, floor or room number`, but not `Service location address` this information now will be floating around without context. Vs on Prod, this information would default to falling under the generic **Contact Info** header

Screenshot 2024-02-26 091548

Screenshot 2024-02-26 091407

Screenshot 2024-02-26 091428

**Comment by aklausmeier** [comment]
@thejordanwood Can you confirm if accordions should have a border like the VBA RO designs? 

(below, pulled Dave's comment above)

> Not matching the design
> 
> The accordions should be the borderless variant.

**Comment by jilladams** [comment]
We chatted about Laura's a11y feedback notes in scrum this morning. There are 2 pieces: 

1. **Appt phone number needs to be right after Appt header.** Agreed, and we want to handle this in this PR.

2. **Header level:** Semantically changing header levels doesn’t fit in the current ticket but makes sense and is a good idea. 
  * To get it done: Needs design review, signoff, and Sketch update 
  * Jordan would like to keep this out of current PR, to make sure we take a bigger look and don’t miss other tweaks we need

I'm working today on scheduling a bigger design/FE call to review the situation with service location accordions across VBA, VAMC, VA Police, (which @laflannery will be invited to) so we can touch on #2 in that meeting as well. (Scrum notes in Trello for reference: https://trello.com/c/jDLQgJ8m/385-service-location-accordions-numbers-spacing)

**Comment by thejordanwood** [comment]
@davidmpickett After some discussion with @aklausmeier, we've decided that VBA accordions should also have borders to stay consistent with the accordions on VAMC pages. 

@eselkin Can borders be added to the VBA pages?

**Comment by eselkin** [comment]
@thejordanwood Yes, VBA pages can have borders on the accordion items. I think you mean the gray (? colorblind) borders. Right?

**Comment by eselkin** [comment]
@davidmpickett 

> While poking around, I noticed a few other things
> 
> **Requested change to content:**
> 
> * When by appointment only is selected, the displayed text should be "Visit our office, ~with an~ by appointment only"
> 
**Changed**

> **Not matching the design**
> 
> * The accordions should be the borderless variant.

**Changing VBA to bordered as well**

> 
> **This isn't good, but I don't know how to fix it**
> 
> * If an editor has entered `building name/number` or `wing, floor or room number`, but not `Service location address` this information now will be floating around without context. Vs on Prod, this information would default to falling under the generic **Contact Info** header
> 
Right now if there's no clinic name but there is subsequent data, we could default to something like:
`Location Info`
In the header space for that section.

**Comment by thejordanwood** [comment]
@eselkin Yes, I meant the gray borders. Here's [an example of the bordered accordions](https://design.va.gov/components/accordion#bordered) in the design system. Thanks!

**Comment by eselkin** [comment]
@thejordanwood finally got the borders on VBA. It had bordered on the va-accordion but it's the items that also need that attribute, but the design website doesn't specify it. It should be up in an hour on https://web-pr15622-xptmfsnjoodldn85nzgkrgljqtfbz24b.ci.cms.va.gov/cleveland-va-regional-benefit-office/locations/cleveland-va-regional-benefit-office/

So, tomorrow.

**Review by laflannery — APPROVED** [review]
Approved!

**Comment by thejordanwood** [comment]
Approved by me!

**Review by thejordanwood — APPROVED** [review]

**Comment by Andrew565** [comment]
PR has failing required tests, opting not to review at this time. - Tier 2/FE COP

**Comment by eselkin** [comment]
@Andrew565 It's going to have failing tests until CMS has merged in the Service Location Paragraphs from their PR/integration branch.

**Comment by jilladams** [comment]
@maxx1128 : reminder re: review and I'm going to go ahead and ping Platform FE just for safety, since this needs to merge by EOD Monday if the CMS deploy goes well.

**Comment by jilladams** [comment]
@eselkin actually: should I wait til CI is passing?

**Comment by eselkin** [comment]
Platform likes to review only after tests pass, but since this is dependent on CMS merging SL it's probably ok to ping them now.

**Comment by jilladams** [comment]
ugh right, I keep forgetting that. Thank you!

**Review by harshil1793 — APPROVED** [review]

**Review by maxx1128 — CHANGES_REQUESTED** [review]
One small concern to address before I think it's good.

File: src/site/facilities/service_address.drupal.liquid [hunk 15]
```diff
@@ -1,3 +1,12 @@
+{% if serviceLocationAddress.fieldUseFacilityAddress and facilityAddress %}
+  {% assign hasAddress = true %}
+  {% assign addressData = facilityAddress %}
+{% elsif location.entity.fieldServiceLocationAddress %}
+  {% assign hasAddress = true %}
+  {% assign addressData = location.entity.fieldServiceLocationAddress.entity.fieldAddress %}
+{% endif %}
```
[the remark this answers is not in the document]
Comment (eselkin) [reply src/site/facilities/service_address.drupal.liquid:7]: ok, will add that

[the remark this answers is not in the document]
Comment (eselkin) [reply src/site/facilities/service_address.drupal.liquid:7]: Changed

**Review by maxx1128 — APPROVED** [review]

**Pull request merged**